client/data/transactions/resolvers.js

```diff
@@ -19,6 +19,17 @@ import {
 } from './actions';
 import { formatDateValue } from 'utils';
 
+function getTimeZone() {
```

Comment:
I found out that we can use `moment(new Date()).format('Z')`, and it should return the formatted timezone offset as we try to format it here (+02:00).
Can you give it a try?